hardhat.config.ts

```diff
@@ -232,7 +232,7 @@ const config: HardhatUserConfig = {
   },
 
   paths: {
-    tests: './{test,plugins/deployment_manager/test}',
+    tests: '.test',
```

Comment:
Whoa, big breaking change!

Reply:
you mean it's a breaking change that `'./{test,plugins/deployment_manager/test}'` stopped working with Hardhat?

Reply:
yes